### DISCUSSION TITLE
Change text color in rowAnnotation?

### DISCUSSION THREAD

**Discussion by lorenzoFabbri** [jokergoo/ComplexHeatmap#770] [Q&A]
I am trying to change text color in `rowAnnotation`, so that each variable with the same name has the same color. I tried to create a vector of the form `"variable1" = "color1"` and pass this vector to `gp = gpar(col = col)`, but the colors do not match (i.e., same color for different variable names). What am I doing wrong? Thanks.

**Comment by jokergoo** [comment]
Can you give me an example?
